src/components/v2v3/shared/FundingCycleConfigurationDrawers/NftDrawer/MaxSupplyFormItem.tsx

```diff
@@ -1,20 +1,16 @@
 import { t } from '@lingui/macro'
 import { Form, Switch } from 'antd'
+import { useWatch } from 'antd/lib/form/Form'
+import useFormInstance from 'antd/lib/form/hooks/useFormInstance'
 import FormattedNumberInput from 'components/inputs/FormattedNumberInput'
 import TooltipLabel from 'components/TooltipLabel'
 
-import { useState } from 'react'
+import { NftFormFields } from './NftRewardTierModal'
 
-export default function MaxSupplyFormItem({
-  value,
-  onChange,
-}: {
-  value: number | undefined
-  onChange: (value: number | undefined) => void
-}) {
-  const [limitedSupplyEnabled, setLimitedSupplyEnabled] = useState<boolean>(
-    Boolean(value),
-  )
+export default function MaxSupplyFormItem() {
+  const form = useFormInstance<NftFormFields>()
```

Comment:
Approving, but make sure you use `Form.useFormInstance(...`

Reply:
Got it. Done